packages/core/src/vector-store/index.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * should compatible with npm:pg and npm:postgres
```

Comment:
I think in the future, there could have sqlite vector store, so type should also comptatable with node:sqlite sqlite3 better-sqlite3

Reply:
https://github.com/run-llama/LlamaIndexTS/pull/1184